internal/sdkv2provider/resource_cloudflare_access_mutual_tls_certificate_test.go

```diff
@@ -43,7 +42,6 @@ func testSweepCloudflareAccessMutualTLSCertificate(r string) error {
 		}
 	}
 
-	zoneID := os.Getenv("CLOUDFLARE_ZONE_ID")
```

Comment:
similar to https://github.com/cloudflare/terraform-provider-cloudflare/pull/2547/files#r1255021871 - this is required for the API call below it.